src/IIIFPresentation/Models/API/Collection/FlatCollection.cs

```diff
@@ -1,37 +1,38 @@
 ﻿using System.Text.Json.Serialization;
+using IIIF.Presentation.V3.Strings;
 
-namespace Models.Response;
+namespace Models.API.Collection;
 
 public class FlatCollection
```

Comment:
Is this a 'normal' IIIF manifest? If so, can classes from iiif-net nuget packages be used?

Reply:
no, this has additional values based on the IIIF-CS-Show-Extras header